### PR TITLE
fix: cannot paste the data in chrome

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* related issue(#1363)

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: packages/toast-ui.grid/src/view/clipboard.tsx [hunk 1]
```diff
@@ -109,12 +109,12 @@ class ClipboardComp extends Component<Props> {
...
     if (html && html.indexOf('table') !== -1) {
       // step 1: Append copied data on contenteditable element to parsing correctly table data.
-      el.textContent = html;
+      el.innerHTML = html;
       // step 2: Make grid data from cell data of appended table element.
       const { rows } = el.querySelector('tbody')!;
       data = convertTableToData(rows);
       // step 3: Empty contenteditable element to reset.
-      el.textContent = '';
+      el.innerHTML = '';
```
Comment (shiren) [inline packages/toast-ui.grid/src/view/clipboard.tsx:117]: 내용상으로는 text만 가져가느냐 html을 가져가느냐의 문제로 작지 않은 변화인데 사이드 이팩트는 없어요?

**Review by shiren — APPROVED** [review]
수고하셨습니다.

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

File: packages/toast-ui.grid/src/view/clipboard.tsx (hunk 1 above)
Comment (js87zz) [reply packages/toast-ui.grid/src/view/clipboard.tsx:117]: 넵 붙여넣기 시에 데이터를 어떻게 가져올지인대 확인하였습니다~!

**Pull request merged**